config.json

```diff
@@ -193,6 +193,18 @@
         "equality",
         "strings"
       ]
+    },
+    {
+      "slug": "diamond",
+      "uuid": "5bbd251d-b2eb-44a1-a9a6-8543afb7511c",
+      "core": false,
```

Comment:
If it's not `core` then it should have an `unlocked_by` right? 🤔

Reply:
Yes

Reply:
Which exercise should it be unlocked by? Or are you saying it needs to just be marked as core?

Reply:
@jvarness I'm going to suggest making it `core` after Anagram. That means setting core as false and unlocked as null.

@devkabiir Thoughts?